File: 김강민/Month_10/1009/1789.py
import sys
input = sys.stdin.readline

S = int(input())

total = 0
arr = []
i = 0

while total <= S:
    if i == 0:
        i += 1
        total += i
    else:
        arr.append(i)
        i += 1
        total += i


print(len(arr))

File: 김강민/2023/Month_10/1009/1789.js
let input = require('fs')
    .readFileSync('/dev/stdin')
    .toString()
    .trim()
    .split('\n')
    .map((e) => e.split(' ').map(Number));

let s = input[0];

let total = 0;
let arr = [];
let cnt = 0;

while (total <= s) {
    if (cnt === 0) {
        cnt += 1;
        total += 1;
    } else {
        arr.push(cnt);
        cnt += 1;
        total += cnt;
    }
}

console.log(arr.length);
